jetstream/tests/integration/test_experimenter_api.py

```diff
@@ -0,0 +1,61 @@
+import datetime as dt
+
+import pytest
+import pytz
+from metric_config_parser.experiment import Branch, Experiment
+
+from jetstream.experimenter import ExperimentCollection
+
+
+@pytest.fixture
+def experiment_collection():
+    try:
+        collection = ExperimentCollection.from_experimenter()
+        return collection
+    except Exception as e:
+        pytest.fail(f"Failed to fetch experiment collection: {e!s}")
+
+
+def test_from_experimenter(experiment_collection):
+    for experiment in experiment_collection.experiments:
+        assert isinstance(experiment, Experiment)
+        for branch in experiment.branches:
+            assert isinstance(branch, Branch)
```

Comment:
I don't think you need to explicitly check the branches -- if the experiment is a valid `Experiment` instance then the branches should be `Branch` instances as well, right?

Otherwise good test! Gets the experiment collection from the API and validates that it fits our expected data structure.